clients/python/tests/conftest.py

```diff
@@ -12,7 +12,7 @@
 from unittest.mock import Mock, patch
 
 import pytest
-import requests
+import requests  # type: ignore[import-untyped,unused-ignore]
```

Comment:
Ideally we should add `types-requests` to the dev dependencies group just under `types-python-dateutil`, but I hit dependency resolution errors with poetry when trying to add it.